### ISSUE TITLE
Profile page UI

### ISSUE THREAD

**Issue by svperfecta** [madglory/nowplayus#196]
We need a little more refactoring on the profile pages.  All the variable height boxes make the page look funny.  Suggestion:
- Limit recent games to 4 or 8
- Show all Accounts for all players.  If a player doesn't have one linked, grey it out
- Fix the height of the profile box

**Comment by erkattak** [comment]
This good now?

**Comment by sethlouey** [comment]
No, this still needs work. It's all over the place. I sill think we need a set height on each container and remove X-height for mobile. If there are no recent games then let's show that and prompt users to join games or schedule. Also need to add a couple more networks so that might change the height of that container
